cypress/integration/ui/about_page.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('about page', () => {
+    before(function () {
+        cy.visit('/about')
+    })
+
+    it('loads successfully', () => {
+        cy.url().should('include', '/about')
+    })
+
+    it('contains appropriate number of images ', () => {
+        cy.get('div')
+            .find('img').should('have.length', 7)
+    })
+
+    it('contains WMD Software logo', () => {
+        cy.get('.my-24 > :nth-child(1) > .mt-8 > [slot="description"]')
```

Comment:
I really don't like idea of testing things basing on css styles. `.my-24` is for margin, not a place in code. Maybe you should add some classes into Svelte templates?